src/main/flush.ts

```diff
@@ -0,0 +1,166 @@
+import fs from 'fs';
+
+export enum FilterTypeEnum {
+  PEAK = 'PK', // ["PK",True,True]
+  LPQ = 'LPQ', // ["LPQ",False,True]
+  HPQ = 'HPQ', // ["HPQ",False,True]
+  BP = 'BP', // ["BP",False,True]
+  LS = 'LS', // ["LS",True,False]
+  HS = 'HS', // ["HS",True,False]
+  NO = 'NO', // ["NO",False,True]
+  AP = 'AP', // ["AP",False,True]
+  LSC = 'LSC', // ["LSC",True,True]
+  HSC = 'HSC', // ["HSC",True,True]
+  BWLP = 'BWLP', // ["BWLP",False,True]
+  BWHP = 'BWHP', // ["BWHP",False,True]
+  LRLP = 'LRLP', // ["LRLP",False,True]
+  LRHP = 'LRHP', // ["LRHP",False,True]
+  LSCQ = 'LSCQ', // ["LSCQ",True,True]
+  HSCQ = 'HSCQ', // ["HSCQ",True,True]
+}
+
+interface IFilter {
+  frequency: number;
+  gain: number;
+  type: FilterTypeEnum;
+  quality: number;
+}
+
+export interface IState {
+  isEnabled: boolean;
+  preAmp: number;
+  filters: IFilter[];
+}
+
+const FIXED_FREQUENCIES = [
+  32, 64, 125, 250, 500, 1000, 2000, 4000, 8000, 16000,
+];
+
+export const DEFAULT_FILTER: IFilter = {
+  frequency: 1000,
+  gain: 0,
+  quality: 1,
+  type: FilterTypeEnum.PEAK,
+};
+
+const DEFAULT_FILTERS: IFilter[] = FIXED_FREQUENCIES.map((f) => {
+  return {
+    frequency: f,
+    gain: 0,
+    quality: 1,
+    type: FilterTypeEnum.PEAK,
+  };
+});
+
+const DEFAULT_STATE: IState = {
```

Comment:
Default state sets all gains to 0, qualities to 1, filters to peak filters, and displays the fixed 10-band frequencies we decided upon.